src/aura/BGE_DonationSelector/BGE_DonationSelector.cmp

```diff
@@ -117,36 +117,38 @@
         <!--END UNPAID PAYMENTS CARDS-->
 
         <!--START OPEN OPPS CARDS-->
-        <aura:iteration var="donation" items="{!v.openOpportunities}">
+        <aura:iteration var="opportunityWrapper" items="{!v.openOpportunities}">
             <lightning:layoutItem largeDeviceSize="4" mediumDeviceSize="6"
                                   size="12"
                                   padding="around-small">
-                <lightning:card title="{!donation.Name}" iconName="standard:opportunity"
-                                class="{!'slds-card_boundary sameCardHeight' + if(equals(donation.Id, v.selectedDonation.Id), ' selectedDonationCard')}">
+                <lightning:card title="{!opportunityWrapper.opportunity.Name}" iconName="standard:opportunity"
+                                class="{!'slds-card_boundary sameCardHeight' + if(equals(opportunityWrapper.opportunity.Id, v.selectedDonation.Id), ' selectedDonationCard')}">
 
                     <aura:set attribute="footer">
                         <div class="selectedDonationBody">
                             <lightning:layout multipleRows="true">
+                                <aura:if isTrue="{!not(opportunityWrapper.hasPayments)}">
+                                    <lightning:layoutItem size="12" padding="around_xx-small"
```

Comment:
Just curious, why did you bring the lightning:layoutItem inside the aura:if?

Reply:
NM I think I get it - you are bringing an additional aura:if condition around "hasPayments", not moving the condition.

Reply:
Yep, exactly - the layout should be unchanged.